### PR TITLE
Various bugfixes and logic error fixes

### PR DESCRIPTION
**Pull request by glensc**
See commit messages for details.

### CODE REVIEW

**Comment by glensc** [comment]
@twolaw I went over the code visually and spotted some problems that decided to address.

**Comment by glensc** [comment]
@twolaw and yet again it remains a mystery to me how can you run the code without required imports like `trakt.errors` :)

**Comment by twolaw** [comment]
> @twolaw and yet again it remains a mystery to me how can you run the code without required imports like `trakt.errors` :)

`import trakt` does the job.

and `import trakt.movies` `import trakt.tv` are needed just after to initialize something i don't remember exactly.

**Comment by glensc** [comment]
PyCharm is indicating this as an error. and doesn't hurt to import to indicate intent.

**Comment by twolaw** [comment]
> 
> 
> and doesn't hurt to import to indicate intent.

Can you rephrase this please ?

**Comment by twolaw** [comment]
`Run m.mark_as_seen for py<3.6 under requests_cache.disabled` can be done also for tvshow section.

**Comment by glensc** [comment]
08f5f3c

**Review by twolaw — APPROVED** [review]
